Sources/MapboxCoreNavigation/DistanceFormatter.swift

```diff
@@ -56,6 +66,15 @@ public struct RoundingTable {
         } ?? thresholds.last!
     }
     
+    /**
+     :no-doc:
```

Comment:
FYI, jazzy only recognizes this spelling:

```suggestion
     :nodoc:
```

In practice, though, the documentation gets hidden anyways because of the `:nodoc:`s on `RoundingTable` and `RoundingTable.Threshold`.

Reply:
ah oops. thanks for the catch.